pub-kube-config-files/publish-availability-monitor.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: publish-availability-monitor
+  labels:
+    app: publish-availability-monitor
+    visualize: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: publish-availability-monitor
+  template:
+    metadata:
+      labels:
+        app: publish-availability-monitor
+        version: "latest"
+        visualize: "true"
+    spec:
+      containers:
+      - name: publish-availability-monitor
+        image: coco/publish-availability-monitor:v0.24.17-k8s-version-rc7
+        imagePullPolicy: Always
+        resources:
+          limits:
+            memory: 512Mi
+        env:
+        - name: KAFKA_TOPIC
+          value: "PreNativeCmsPublicationEvents"
+        - name: KAFKA_PROXY_HOST
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: kafka.proxy.url
+        - name: QUEUE_ADDR
+          value: "http://kafka:8082"
+        - name: CONTENT_URL
+          value: "/__document-store-api/content/"
+        - name: INTERNAL_COMPONENTS_URL
+          value: "/__document-store-api/internalcomponents/"
+        - name: LISTS_URL
+          value: "/__document-store-api/lists/"
+        - name: NOTIFICATIONS_URL
+          value: "/__notifications-rw/content/notifications?type=all"
+        - name: NOTIFICATIONS_PUSH_URL
+          value: "/content/notifications-push?monitor=true"
+        - name: LISTS_NOTIFICATIONS_URL
+          value: "/__list-notifications-rw/lists/notifications"
+        - name: LISTS_NOTIFICATIONS_PUSH_URL
+          value: "/lists/notifications-push?monitor=true"
+        - name: METHODE_ARTICLE_VALIDATION_URL
+          value: "https://k8s-delivery-upp-eu.ft.com/__methode-article-mapper/map"
```

Comment:
Please leave a todo in here, that we'll have to template these when we extract this to helm